content/posts/weekly/June-2022.md

```diff
@@ -32,3 +32,11 @@ weight: 61
   ([packit-service#1556](https://github.com/packit/packit-service/pull/1556))
 - A link to Copr build logs was updated: it now points to a place where logs are available once a build starts.
   ([packit-service#1554](https://github.com/packit/packit-service/pull/1554))
+
+## Week 25 (June 21st - June 27th)
+
+- Packit Bash completion file is no longer needlessly executable. ([packit#1634](https://github.com/packit/packit/pull/1634))
+- Packit now works with Bodhi 5 and Bodhi 6 authentication mechanism. ([packit#1629](https://github.com/packit/packit/pull/1629))
+- Git ref name that Packit works with during `propose-downstream` is now made more obvious in logs. ([packit#1626](https://github.com/packit/packit/pull/1626))
+- We have fixed the incorrect displaying of `propose downstream` results in our dashboard. ([packit-service#1553](https://github.com/packit/packit-service/pull/1553))
+- There is a new check for git projects that are allowed to use a custom Copr project. There will be a better integration in the form of a new config field in Copr settings that Packit can use. In the meantime, the mapping is defined and maintained by the Packit team. Let us know if you need a project to be allowed. ([packit-service#1556](https://github.com/packit/packit-service/pull/1556))
```

Comment:
```suggestion
```

This one is already mentioned in the previous week.

Reply:
Oh thanks!